Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -2002,6 +2130,25 @@ public function testGetUrlsafeIdentifier_GivenAdmin_Bar()
         $this->assertSame(1234567, $this->twigExtension->getUrlsafeIdentifier($entity, $this->adminBar));
     }
 
+    // TODO: remove this method for next major release only if PHP 5.3 support will be dropped
```

Comment:
No `TODO` allowed on the code.

Just move you sentence below the first on on the phpdoc of `buildTwigLikeUrl`. :+1: